examples/hf_trainer_api/image_classification/requirements.txt

```diff
@@ -4,5 +4,5 @@ torch>=1.5.0
 torchvision>=0.6.0
 datasets
 evaluate
-transformers==4.28.1

```

Comment:
I would suggest verifying that this still works with `transformers >= 4.28.1`.  Hit an issue with a different hf example today due to changes in a recent release, I think `4.29.2`

Reply:
Maybe something like between `>=4.28.0,<2.29.0`? I just am trying to be permissive if at all possible